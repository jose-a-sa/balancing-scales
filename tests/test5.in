#ScaleName,Left,Right
A1,B1,B2
B1,B3,B4
B2,B5,10
B3,4,1
B4,2,B6
B5,3,2
B6,B7,5
B7,6,12
