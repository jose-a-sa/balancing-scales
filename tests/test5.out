A1,0,186
B1,94,0
B2,3,0
B3,0,3
B4,49,0
B5,0,1
B6,0,20
B7,6,0